tools/container/runtime/containerd/containerd.go

```diff
@@ -84,13 +85,7 @@ func Flags(opts *Options) []cli.Flag {
 func Setup(c *cli.Context, o *container.Options, co *Options) error {
 	log.Infof("Starting 'setup' for %v", c.App.Name)
 
-	cfg, err := containerd.New(
-		containerd.WithPath(o.Config),
-		containerd.WithConfigSource(containerd.CommandLineSource(o.HostRootMount)),
-		containerd.WithRuntimeType(co.runtimeType),
-		containerd.WithUseLegacyConfig(co.useLegacyConfig),
-		containerd.WithContainerAnnotations(co.containerAnnotationsFromCDIPrefixes()...),
-	)
+	cfg, err := getConfigurer(o, co)
```

Comment:
nit: can we choose a name that is more specific here? `getConfigurer` sounds a bit too vague to me